genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/RightPanel.tsx

```diff
@@ -106,15 +104,31 @@ export default function RightPanel(): JSX.Element {
           {experiences && experiences.length > 0 ? (
             <Flex direction="column" gap="size-200">
               <View paddingX="size-200" paddingY="size-100">
-              <Picker
-                label="Select a claim library"
-                width="100%"
-                onSelectionChange={handleClaimsLibrarySelection}
-              >
-                {TEST_CLAIMS.map(library => (
-                  <Item key={library.id}>{library.name}</Item>
-                ))}
-              </Picker>
+              <Flex direction="row" justifyContent="space-between" alignItems="center" marginBottom="size-100">
```

Comment:
app got kind of busy, so i added dividers, headers and had to adjust the spacing a bit